app.js

```diff
@@ -308,6 +309,22 @@ define(function(require) {
 					i18n: self.i18n.active().webhooks.webhookEdition.customDataLabels
 				});
 
+				customHeaders = _
+					.chain(webhookData.webhookDetails)
+					.get('custom_http_headers', {})
+					.transform(function(data, value, key) {
+						if (protectedCustomData.indexOf(key) < 0) {

```

Comment:
This check is necessary only for custom data, not custom headers. Please remove it.